specification/cognitiveservices/data-plane/AnomalyDetector/preview/v2.0-preview/MultidimensionalRootCauseAnalysis.json

```diff
@@ -0,0 +1,2031 @@
+          "__SUM__",
+          "0.009",
+          "9",
+          "1000"
+        ],
+        [
+          "DE",
+          "__SUM__",
+          "0.011",
+          "11",
+          "1000"
+        ],
+        [
+          "CA",
+          "__SUM__",
+          "0.01",
+          "1",
+          "1000"
+        ],
+        [
+          "__SUM__",
+          "__SUM__",
+          "1",
+          "1000",
+          "1000"
+        ]
+      ]
+    },
+    {
+      "partitionIdentity": "PartitionIdentity5",
+      "timestamp": "2021-11-27T00:00:00Z",
+      "data": [
+        [
+          "CN",
+          "__SUM__",
+          "0.299",
+          "299",
+          "1000"
+        ],
+        [
+          "CN",
+          "ios",
+          "0.31",
+          "310",
+          "1000"
+        ],
+        [
+          "CN",
+          "Android",
+          "0。001",
+          "1",
+          "1000"
+        ],
+        [
+          "US",
+          "__SUM__",
+          "0.401",
+          "401",
+          "1000"
+        ],
+        [
+          "JP",
+          "__SUM__",
+          "0.01",
+          "10",
+          "1000"
+        ],
+        [
+          "DE",
+          "__SUM__",
+          "0.01",
+          "10",
+          "1000"
+        ],
+        [
+          "CA",
+          "__SUM__",
+          "0.01",
+          "1",
+          "100"
+        ],
+        [
+          "__SUM__",
+          "__SUM__",
+          "1",
+          "1000",
+          "1000"
+        ]
+      ]
+    }
+  ]
+}
+    },
+    "MultiDimRCADataMeta": {
+      "required": [
+        "createdDateTime",
+        "dataSize",
+        "lastUploadedDateTime",
+        "partitionIdentity",
+        "timestamp"
+      ],
+      "type": "object",
+      "properties": {
+        "partitionIdentity": {
+          "description": "Multidimensional root cause analysis partition identity",
+          "type": "string"
+        },
+        "dataSize": {
+          "format": "int32",
+          "description": "Multidimensional root cause analysis data size in the partition identity",
+          "type": "integer"
+        },
+        "timestamp": {
+          "format": "date-time",
+          "description": "Multidimensional root cause date time of the data",
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Multidimensional root cause data created time",
+          "type": "string"
+        },
+        "lastUploadedDateTime": {
+          "format": "date-time",
+          "description": "Multidimensional root cause data last time uploaded",
+          "type": "string"
+        }
+      }
+    },
+    "MultiDimRCADataMetaListResponse": {
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "@nextLink": {
+          "description": "Link of next page, only provided when there are next pages",
+          "type": "string",
+          "readOnly": true
```

Comment:
There is no need to define `nextLink` as `readOnly: true` as this schema is only used in a response.